src/main/java/de/tum/in/www1/artemis/web/rest/TextAssessmentResource.java

```diff
@@ -336,7 +337,7 @@ public TextAssessmentResource(AuthorizationCheckService authCheckService, TextAs
     }
 
     /**
-     * Retrieve the result of an example assessment, only if the user is an instructor or if the submission is used for tutorial purposes.
+     * GET exercise/:exerciseId/submission/:submissionId/example-result : Retrieve the result of an example assessment, only if the user is an instructor or if the submission is used for tutorial purposes.
```

Comment:
2 inconsistencies:

1) sometimes we use "submission", sometimes "submissions" --> I think it should be consistently plural.
2) sometimes we use "text-submissions", sometimes "submissions" --> should be consistent throughout the whole Server API! Either we consistently always use submissions for all sub-types. Or we distinguish between the sub types in most cases (where not only one REST call can be used to handle all cases)

In general, I would prefer one REST call for multiple sub types if this is feasible, because this will lead to more consistent behavior and will also reduce much duplicated code.